### DISCUSSION TITLE
Printer Profile for Prusa XL Single Tool / Multi Tool

### DISCUSSION THREAD

**Discussion by Cloud-Tickler** [OrcaSlicer/OrcaSlicer#2168] [Ideas]
Hello together,

is there a chance adding profiles for the Prusa XL single tool and multi tool in OrcaSlicer? As far as I understand, I can't use the custom profiles or the MK4 profile since the firmware is neither Marlin or Klipper nor MK4.
Beside having the easier to use interface, with the amount of stringing that is happening on most Prusa XL, there is a high demand to generate retraction tests and other calibration prints:-).

Thank you so much.

Regards

**Comment by Topless-Stang** [comment]
I have a 5T XL...was hoping it would be supported by now. Hope it gets added soon.

**Comment by fscheps** [comment]
I hope this is coming soon 😉  Prusa Slicer supports exporting the Config Bundle, but Orca Slicer (which is based on Prusa Slicer) doesn't have this option :/

**Comment by freeformz** [comment]
Single tool support should be out in the next release: https://github.com/SoftFever/OrcaSlicer/pull/5511

**Comment by SupItsSaige** [comment]
PLZ 5T head version I have been waiting so patiently for an update. I can't wait to compare the two!

**Comment by freeformz** [comment]
@SupItsSaige Nightly builds now have this: https://github.com/SoftFever/OrcaSlicer/releases/tag/nightly-builds
